src/utils/url.ts

```diff
@@ -9,6 +9,14 @@ export const venueInsideUrl = (venueId: string) => {
   return `/in/${venueId}`;
 };
 
+export const adminv3VenueUrl = (venueId: string) => {
+  return `/admin-v2/venue/${venueId}`;
+};
+
+export const adminV3SettigsUrl = (venueId: string) => {
+  return `/admin-v2/advanced-settings/${venueId}`;
+};
+
```

Comment:
Naming inconsistency, same comment as https://github.com/sparkletown/sparkle/pull/1511/files#r647324726

There's probably good reason why the function names have 3 while routes 2, so electing not to go there.

Reply:
New naming convention in: https://github.com/sparkletown/sparkle/pull/1511/commits/2afccca403af90d38dca5a81a91041709b2afac3